static/truewiki/truewiki.css

```diff
@@ -35,8 +35,16 @@ body > nav {
 	height: 32px;
 	margin: 0px auto;
 	min-height: 32px;
-	overflow: hidden;
+	/* If the breadcrumbs are longer than the width of the screen, allow horizontal scrolling */
+	overflow: scroll;
+    white-space: nowrap;
+    scrollbar-width: none;
```

Comment:
some whitespaces fixes, and can you put it on alphabet? (so `scrollbar-width` should be higher)